packages/common/driver-definitions/src/storage.ts

```diff
@@ -381,7 +381,9 @@ export interface IDocumentService extends IEventProvider<IDocumentServiceEvents>
 	/**
 	 * Subscribes to the document delta stream
 	 */
-	connectToDeltaStream(client: IClient): Promise<IDocumentDeltaConnection>;
+	//* TODO
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	connectToDeltaStream(client: IClient, steps: any[]): Promise<IDocumentDeltaConnection>;
```

Comment:
here it's steps: any[], but implementation has same 2nd argument as status?: { steps: string[] }